releases/release-1.30/README.md

```diff
@@ -0,0 +1,121 @@
+| 1.30.0-alpha.3 released                                                                | Branch Manager                | Thursday 7th March 2024                                                                                            | Week 9   |                                                        |
+| Deprecations and Removals blog published                                               | Comms                         | Thursday 7th March 2024                                                                                            | week 9   |                                                        |
+| **Preparing for Feature blogs review — Initiating outreach**                           | Comms                         | Monday 11th March 2024                                                                                             | week 10  |                                                        |
+| Docs deadline — PRs ready for review                                                   | Docs Lead                     | Tuesday 12th March 2024                                                                                            | Week 10  |                                                        |
+| **KubeCon EU**                                                                         |                               | Tuesday 19th - Friday 22nd March 2024                                                                               | week 11  |                                                        |
+| **Begin [Burndown]** (Monday, Wednesday, and Friday meetings)                          | Lead                          | Monday 18th March 2024                                                                                             | week 11  |                                                        |
+| **[Test Freeze]**                                                                      | Branch Manager                | [01:00 UTC Wednesday 20th March 2024 / 18:00 PDT Tuesday 19th March 2024](https://everytimezone.com/s/e016958f) | week 11  |                                                        |
+| release-1.30 branch created                                                            | Branch Manager                | Tuesday 19th March 2024                                                                                            | week 11  |                                                        |
+| release-1.30 jobs created                                                              | Branch Manager                | Tuesday 19th March 2024                                                                                            | week 11  |                                                        |
```

Comment:
`release-1.30` branch and jobs are created upon cutting the rc.0 release. We should either move rc.0 to March 19 or move those two milestones to March 26